From 2b3b79e5e64d51f027d815f8767443db784abade Mon Sep 17 00:00:00 2001
Message-Id: <2b3b79e5e64d51f027d815f8767443db784abade.1659372189.git.thomas@gllm.fr>
In-Reply-To: <5b01e96eabd14521cfeb33b5ac11495b77f894cc.1659372189.git.thomas@gllm.fr>
References: <5b01e96eabd14521cfeb33b5ac11495b77f894cc.1659372189.git.thomas@gllm.fr>
From: Thomas Guillem <thomas@gllm.fr>
Date: Thu, 21 Jul 2022 17:19:26 +0200
Subject: [PATCH 09/19] contrib: x264: with build with Android NDK25

Cross tools changed from ${ARCH}-linux-android${API}-${TOOL} to
llvm-${TOOL).

Example: aarch64-linux-android21-strings to llvm-strings.
---
 contrib/src/x264/rules.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/contrib/src/x264/rules.mak b/contrib/src/x264/rules.mak
index 26a8cc2fe6..a9f1f44b5a 100644
--- a/contrib/src/x264/rules.mak
+++ b/contrib/src/x264/rules.mak
@@ -49,6 +49,7 @@ ifndef HAVE_DARWIN_OS
 X264CONF += --cross-prefix="$(HOST)-"
 endif
 ifdef HAVE_ANDROID
+X264CONF += --cross-prefix="$(subst ar,,$(AR))"
 # broken text relocations
 ifeq ($(ANDROID_ABI), x86)
 X264CONF += --disable-asm
-- 
2.35.1

